client/dashboard/sites/overview/visibility-card.tsx

```diff
@@ -0,0 +1,103 @@
+import { CircularProgressBar } from '@automattic/components';
+import { __experimentalText as Text } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { lockOutline, published } from '@wordpress/icons';
+import { launch } from '../../components/icons';
+import { isSelfHostedJetpackConnected } from '../../utils/site-types';
+import OverviewCard from '../overview-card';
+import type { Site } from '../../data/types';
+
+const CARD_PROPS = {
+	title: __( 'Visibility' ),
+	trackId: 'visibility',
+};
+
+function getVisibilityURL( site: Site ) {
+	if ( isSelfHostedJetpackConnected( site ) ) {
+		return undefined;
+	}
+
+	return `/sites/${ site.slug }/settings/site-visibility`;
+}
+
+function VisibilityCardUnlaunched() {
+	return (
+		<OverviewCard
+			{ ...CARD_PROPS }
+			icon={ launch }
+			heading={ __( 'Coming soon' ) }
```

Comment:
How about "Finish setup" to align with the site list?

Reply:
Do we want to add URL in the follow-up PR?

Reply:
The description is already "Finish setting up your site" so it might be a bit repetitive. I think keeping it as "Coming soon" is fine? Or reverse it to something like 

heading - Finish setup
description - Your site is hidden from visitors until it is launched

@matt-west WDYT?